src/EventListener/AddFormatListener.php

```diff
@@ -84,11 +88,11 @@ public function onKernelRequest(GetResponseEvent $event)
                 return;
             }
 
-            throw $this->getNotAcceptableHttpException($mimeType);
+            throw $this->getNotAcceptableHttpException($mimeType ?? $requestFormat);
```

Comment:
`$requestAcceptedFormats` right? Thus should be covered by a test (because the error isn't caught now)